src/scenes/concretes/main_menu/main_menu_render.py

```diff
@@ -49,7 +51,7 @@ def render(self) -> None:
             text_rect = text.get_rect(center=self.option_positions[index])
             self._screen.blit(text, text_rect)
 
-        top_text = f"Top: {self.top_score:06}"
+        top_text = f"Top- {self.top_score:06}"
```

Comment:
Please add a space next to Top, f"Top , to make it more symmetrical

Reply:
Done